packages/react-core/src/components/Alert/AlertIcon.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/Alert/styles.css';
+import {
+  CheckCircle,
+  ExclamationCircle,
+  ExclamationTriangle,
+  InfoCircle
+} from '@patternfly/react-icons';
+
+export const incoType = {
+  success: <CheckCircle size="md" />,
+  danger: <ExclamationCircle size="md" />,
+  warning: <ExclamationTriangle size="md" />,
+  info: <InfoCircle size="md" />
+};
+
+const propTypes = {
+  type: PropTypes.oneOf(Object.keys(incoType)).isRequired,
```

Comment:
make this variant to match with Alert/Button we will probably be making a global variant type since most are shared.